src/app/util.clj

```diff
@@ -0,0 +1,4 @@
+(ns app.util)
+
+(defmacro lazy-component [the-sym]
+  `(app.util/lazy-component* (shadow.lazy/loadable ~the-sym)))
```

Comment:
1. ` is the `quote syntax` - it is used to delay evaluation

https://clojure.org/guides/weird_characters#syntax_quote

2. `~` is the `unquote syntax` 
What does it do?
"Syntax unquote, like quote, means that evaluation is not occurring within the syntax quoted form. Unquoting turns off quoting and evaluates an expression inside the syntax quoted expression."

https://clojure.org/guides/weird_characters#unqote